### PR TITLE
Feat/connect iframe

### LINKED ISSUE #3734: Automate connect build in Suite
Stemmed from [this thread](https://github.com/trezor/trezor-suite/pull/3722#discussion_r628054227).

Some parts of connect that are used in Suite are not published to NPM connect package. (Instead they're published to connect.suite.io to satisfy same-origin policies).

In https://github.com/trezor/trezor-suite/pull/3722 we to stoped fetching these parts from connect.suite.io and bundled them in suite directly[1]. To acomplish this we [replicate](https://github.com/trezor/trezor-suite/blob/develop/packages/suite-data/trezor-connect.webpack.js) the build process of connect in suite-data package in the Suite monorepo. This results in the needed parts being generated from the NPM connect package and commited into suite monorepo. This is error prone as when one updates connect dependency in suite they must also remember to rebuild the files in suite-data. It also unnecessarily replicates the webpack config.

Automate this process. Keep in mind that in near future connect will be moved to monorepo as well.

cc @tsusanka @szymonlesisz 

[1] meaning that they're statically resolved in desktop env and hosted on suite.io in web env.

### PR DESCRIPTION
**Pull request by szymonlesisz**
fix https://github.com/trezor/trezor-suite/issues/3734
related to https://github.com/trezor/trezor-suite/issues/3738 and https://github.com/trezor/trezor-suite/issues/2924

Added 2 new packages:

__@trezor/connect-common__ public npm package
TBD future `trezor-connect` shared files/utils.
currently contains only firmware and bridge release, will be extended in near future.
releases from this package are being used in suite-web and desktop build. it is essential to fulfill https://github.com/trezor/trezor-suite/issues/3738

__@trezor/connect-iframe__ private package
TBD future endpoint hosted on `connect.trezor.io` domain.
used to build required `trezor-connect` files during `yarn build:libs` process

Removed `yarn build:connect` process and `suite-data` static build of `connect`.
connect files are now copied from `@trezor/connect-iframe/build`

### CODE REVIEW

File: packages/connect-common/tsconfig.lib.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",
+        "lib": ["dom", "es2015"],
+        "outDir": "./lib",
+        "strict": true,
+        "declaration": true,
+        "resolveJsonModule": true,
+        "noImplicitAny": false,
+        "preserveSymlinks": true, // ts-node: "Unable to require `.d.ts` file." (https://github.com/TypeStrong/ts-node/issues/797)
```
Comment (matejkriz) [inline packages/connect-common/tsconfig.lib.json]: it's invalid syntax to have comment in JSON, what about to switch to js file?

File: packages/connect-iframe/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`suite-*` builds requires self-hosted `trezor-connect` build to communicate with instead of official `connect.trezor.io`.
+
+`trezor-connect` project needs to be moved to monorepo. This project should eventually replace `trezor-connect` [iframe entry point](https://github.com/trezor/connect/blob/develop/src/js/iframe/iframe.js)
+
+`firmware releases` should be provided by suite monorepo `@trezor/connect-common` package.
+
+### Notes
+
+`@trezor/blockchain-link": "*"` and `"@trezor/rollout": "*"` are listed as dependency to prevent `lerna` from calling `build:lib` before the are installed. This will probably change after moving `trezor-connect` to monorepo.
```
Comment (matejkriz) [inline packages/connect-iframe/README.md:25]: nit: maybe they could be moved to `devDependencies` if they are required for build only?

**Review by matejkriz — COMMENTED** [review]
just a two small comments, let's discuss the architecture and naming questions face to face

File: packages/connect-common/tsconfig.lib.json (hunk 1 above)
Comment (szymonlesisz) [reply packages/connect-common/tsconfig.lib.json]: fixed, 13556bfcf12dc5fffb8bd6d1a110321bed6fa17d
fun fact, we have it like that in multiple packages

File: packages/connect-iframe/README.md (hunk 2 above)
Comment (szymonlesisz) [reply packages/connect-iframe/README.md:25]: i guess it doesnt matter. this package is not public so no one will use those dependencies.
moved them f4379a4

Comment (matejkriz) [reply packages/connect-iframe/README.md:25]: yeah, it was a small nit, thanks anyway!

**Review by matejkriz — APPROVED** [review]

**Pull request merged**